### PR TITLE
Include monicas destinations

### PR DESCRIPTION
**Pull request by monicamillad**
I'll travel the world !!

### CODE REVIEW

**Comment by wewark** [comment]
Wal3ana PR w 7rkat 😂

**Comment by monicamillad** [comment]
@wewark  lol xDD l3lmk ana ba2eet git geek :''''''D

File: css/app.css [hunk 1]
```diff
@@ -58,3 +58,15 @@ h2 {
 #barcelona {
     background-color: #800000;
 }
+
+#florida {
```
Comment (wewark) [inline css/app.css:62]: El nas b2a momken t3mlk review w t2olk bla bla 7lw

**Review by wewark — APPROVED** [review]

File: index.html [hunk 2]
```diff
@@ -17,6 +17,18 @@ <h2>Paris</h2>
             <div class="destination" id="barcelona">
                 <h2>Berlin</h2>
             </div>
+
+            <div class="destination" id="florida">
```
Comment (wewark) [inline index.html:21]: aw we7esh

**Review by wewark — CHANGES_REQUESTED** [review]

**Comment by wewark** [comment]
momken arod 3leky mn el email law ana mksl aft7 github :joy:

**Comment by monicamillad** [comment]
@wewark  aw mmkn aklmk whatsapp bdl elfadaye7 deah enta mesh pro khales :P

**Pull request closed without merging**